src/org/gk/elv/InstanceZoomablePathwayEditor.java

```diff
@@ -553,10 +552,11 @@ public Renderable insertInstance(GKInstance instance) {
             Renderable container = pathwayEditor.getRenderable();
             // Don't set container so that the following simpleConvert()
             // will not add the converted instance directly to this container.
-            // Let the insert statments figure this out.
+            // Let the insert statements figure this out.
             //handler.setContainer(container);
             handler.setContainer(null);
             Renderable r = handler.simpleConvert(instance);
+//            Renderable r = InstanceToRenderableConverter.convertToNode(instance, false);
```

Comment:
Please remember to remove your test code before push!

Reply:
Will do!